Description: Fix build failure with GCC 4.7
 This patch fixes a build failure that occurs only when using GCC 4.7 or newer.
Author: Micha Lenk <micha@debian.org>
Origin: upstream, commit:r2146
Bug-Debian: http://bugs.debian.org/672074
Forwarded: not-needed
Applied-Upstream: commit:r2146, 4.3.4
Last-Update: 2012-05-10

--- libgwenhywfar-4.3.3.orig/gui/fox16/libtest.cpp
+++ libgwenhywfar-4.3.3/gui/fox16/libtest.cpp
@@ -22,6 +22,8 @@
 
 #include <gwenhywfar/debug.h>
 
+#include <unistd.h>
+
 
 
 int test1(int argc, char **argv) {
